docs/reference/glossary.md

```diff
@@ -153,6 +153,12 @@ installs Docker on them, then configures the Docker client to talk to them.
 
 *Also known as : docker-machine*
 
+## node
+
+A [node](../swarm/how-swarm-mode-works/nodes.md) is a physical or virtual
+machine running an instance of the Docker Engine in swarm mode. Manager nodes
+perform swarm management and orchestration duties. Worker nodes execute tasks.
+
```

Comment:
mmm, this makes it a little unclear that a manager node can also be a worker node (unless drained.)